management/dbaas/kubernetes_server_test.go

```diff
@@ -104,4 +105,62 @@ func TestKubernetesServer(t *testing.T) {
 		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "invalid field KubernetesClusterName: value '' must not be an empty string")
 		require.Nil(t, unregisterKubernetesClusterOK)
 	})
+
+	t.Run("UnregisterWithForce", func(t *testing.T) {
```

Comment:
let's test it this way
1. register kubernetes cluster
2. create db cluster
3. try to unregister without force and get error
4. unregister kubernetes cluster with force 
5. check that kubernetes cluster is unregistered
6. re-register kubernetes cluster
7. delete db cluster
8. wait until db cluster is deleted
9. unregister kubernetes cluster without force

Reply:
Ok, I will extend this test.

Reply:
@BupycHuk Test is rewrited with this way, but for "8. wait until db cluster is deleted" I am using time.Sleep right now, because I am not sure how to wait properly.  Can you help me? Thanks.

Reply:
Request list in a loop with sleep and break when there is no db cluster